src/app/relay/CreateTagMutation.js

```diff
@@ -18,7 +18,7 @@ class CreateTagMutation extends Relay.Mutation {
       query = Relay.QL`fragment on CreateTagPayload { tagEdge, project_media { log, tags, log_count } }`;
       break;
     case 'project_source':
-      query = Relay.QL`fragment on CreateTagPayload { tagEdge, project_source { tags } }`;
+      query = Relay.QL`fragment on CreateTagPayload { tagEdge, project_source { tags, source { log, log_count } } }`;
```

Comment:
Same here. I'm not sure about this. Should `log` be under `source` or `project_source`?